src/delay_handler.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 #include "request_handler.h"
-const size_t DEFAULT_DELAY = 1000;
+const size_t DEFAULT_DELAY = 1000000;
```

Comment:
Maybe add a comment with the units this is in? or make it `DEFAULT_DELAY_MICROSECONDS` or something?